H2MLauncher.UI/Converters/GameModeConverter.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Globalization;
+using System.Windows.Data;
+
+using H2MLauncher.Core.Settings;
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace H2MLauncher.UI.Converters;
+
+public sealed class GameModeConverter : IValueConverter
+{
+    private Dictionary<string, string>? _gameModeMap = null;
+
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        if (_gameModeMap is null)
+        {
+            ResourceSettings resourceSettings = App.ServiceProvider.GetService<IOptions<ResourceSettings>>()!.Value;
```

Comment:
😱 please never do that, this is a total anti pattern (service locator + service in xaml layer) :D 
I really think this should not be done in a converter but in the view model. Either have an additional property `MapDisplayName` or put it in a class like the `MapPackItem`. Then we can also toggle that with a boolean

Reply:
I did this as a way to get it "working". I couldn't figure out how to get a mapping to work otherwise, as the Converter IS responsible for the mapping visually.